Bindings/Packages/com.virtualmaker.bindings/Runtime/VirtualMaker.Bindings.asmdef

```diff
@@ -11,6 +13,42 @@
     "precompiledReferences": [],
     "autoReferenced": true,
     "defineConstraints": [],
-    "versionDefines": [],
+    "versionDefines": [
+        {
+            "name": "com.unity.ugui",
```

Comment:
copy what flexalon does for textmeshpro. For versions before Unity 6, it's a separate package

Reply:
Added and hopefully used around the relevant parts. 👍

Reply:
Changed to `UNITY_*`